### PR TITLE
Documentation: Created controlled radio example. Meta: Added dist dir…

### PR DESCRIPTION
**Pull request by brianboyko**
Added a controlled radio example to documentation, showing primarily how one would use Vuex actions/getters to control a set of radio buttons.  (Even though the documentation does not use vuex, it uses a method to create the same effect.) 

Also added the dist/ directory to gitignore (seems like an oversight) and added a few more clarifying examples to the primary radio button example. 
screen shot 2018-02-08 at 11 01 05 am

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,4 @@ coverage/
 yarn-debug.log
 npm-debug.log
 .vscode
+dist/
```
Comment (VdustR) [inline .gitignore]: don't ignore this or unpkg link will be dead

**Comment by marcosmoura** [comment]
@brianboyko The documentation uses Vuex. You can use that on your example.
As @VdustR said, do not ignore dist folder.

**Comment by marcosmoura** [comment]
@brianboyko This is a good example, but we also have to add the same example for Checkbox and Switch. Can you do that?

**Comment by Samuell1** [comment]
@brianboyko Any updates to this PR?

**Pull request closed without merging**